community/onboarding-offboarding.md

```diff
@@ -12,39 +12,40 @@ See also [Fresher's Induction](/docs/community/freshers-induction).
 1. Add them to the Google Group. Check the delivery settings.
 1. Add them to the Slack.
 1. Add them to their batch’s private channel.
-1. Add their contact details on https://github.com/kossiitkgp/secrets
-1. Assign their 1:1 group
-1. Add their name to the website
+1. Add their contact details on https://github.com/kossiitkgp/secrets .
+1. Assign their 1:1 group.
+1. Add their name to the website.
 1. Add them to facebook groups (unofficial)
-1. Give access to KOSS’ facebook page
-1. Make them store numbers of everyone in team
+1. Give access to KOSS’s facebook page.
+1. Make them store numbers of everyone in team.
 
 ## How to re-designate Core Team Members as Executives?
 1. Make all Executives the Admins of Slack workspace.
 1. Add all Executive Members and Heads to the `Admins` team on GitHub org. Remove them from the `Newbies` team.
 1. Ask an Owner of kossiitkgp GitHub org to change role of Executive Heads as `Owners`.
 1. Make Executive Heads the Managers of the Google Group.
-1. Update Contacts README on `kossiitkgp/secrets`
-1. Release the names from blog/facebook page
-1. Update the “Members” section on the website
+1. Make all Executives the Admins of the Facebook Page.
```

Comment:
Will it be editor or admin ?

Reply:
Admin -> Can add, promote, demote page roles.
Editor -> Can't do these.

This is written as `Admin`, coz they can add others and be themselves be remove by admins easily.

There might also be grounds for keeping this `editor` instead. Please share your take on this.